pkgs/by-name/pc/pcsc-safenet/package.nix

```diff
@@ -4,23 +4,20 @@
 , autoPatchelfHook
 , dpkg
 , gtk3
-, openssl_1_1
+, openssl
 , pcsclite
 }:
 
 stdenv.mkDerivation rec {
   pname = "pcsc-safenet";
-  version = "10.8.28";
+  version = "10.8.1050";
 
-  debName = "Installation/Standard/Ubuntu-2004/safenetauthenticationclient_${version}_amd64.deb";
+  debName = "safenetauthenticationclient_${version}_amd64.deb";
 
   # extract debian package from larger zip file
-  src =
-    let
-      versionWithUnderscores = builtins.replaceStrings ["."] ["_"] version;
-    in fetchzip {
-      url = "https://www.digicert.com/StaticFiles/SAC_${versionWithUnderscores}_GA_Build.zip";
-      hash = "sha256-7XWj3T9/KnmgQ05urOJV6dqgkAS/A2G7efnqjQO2ing=";
+  src = fetchzip {
+      url = "https://www.globalsign.com/en/safenet-drivers/USB/10.8/GlobalSign-SAC-Ubuntu-2204.zip";
```

Comment:
This URL does not include the full version. Therefore it is most likely not suitable.